integration/spark2/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -432,7 +432,8 @@ object AlterTableUtil {
       "LOCAL_DICTIONARY_INCLUDE",
       "LOCAL_DICTIONARY_EXCLUDE",
       "LOAD_MIN_SIZE_INMB",
-      "RANGE_COLUMN")
+      "RANGE_COLUMN",
+      "SORT_SCOPE")
```

Comment:
please do the validation for sort_scope in alter set. For reference you can check the alter SET for local_dictionary or CACHE_COLUMNS

Reply:
add test cases for those validations also